addons/wowl/static/tests/action_manager/action_manager_tests.ts

```diff
@@ -49,7 +42,14 @@ QUnit.module("Action Manager Service", {
 
     serviceRegistry.add(actionManagerService.name, actionManagerService);
     serviceRegistry.add(notificationService.name, notificationService);
-    serviceRegistry.add("router", makeFakeRouterService());
+    serviceRegistry.add(
+      "router",
+      makeFakeRouterService({
+        redirect: (url) => {
+          assert.step(url);
```

Comment:
typically we don't do assertions in the "configuration code", because later on, when someone will write a test, he might get into this function and thus have an assert.steps coming from nowhere. So long story short, you should do that only in the test that tests it.